dev/data/fineweb.py

```diff
@@ -22,18 +22,22 @@
 import os
 import argparse
 import multiprocessing as mp
+
 import numpy as np
 import tiktoken
 from datasets import load_dataset
 from tqdm import tqdm
-import argparse
+
+from transformers import AutoTokenizer
```

Comment:
delete dep

Reply:
as in if we decide not to use AutoTokenizer (due to its black-box nature)?